content/src/scripts/libs/contentDark.js

```diff
@@ -0,0 +1,90 @@
+// themeList.js
+
```

Comment:
State the purpose of this file in comments and why it was required to be executed as a different script. **Briefly**